### PR TITLE
Fix: TypeError: BpmnModdle is not a constructor

### PR DESCRIPTION
**Pull request by corocoto**
## Description:
This PR updates [`bpmn-moddle`](https://github.com/bpmn-io/bpmn-moddle) package to the latest version. And refactors the `cleanDI` function at the test file and `AutoLayout`'s `layoutProcess` method.

## Related issues:
* https://github.com/bpmn-io/bpmn-auto-layout/issues/18
* https://github.com/bpmn-io/bpmn-auto-layout/issues/22

### CODE REVIEW

File: test/AutoLayoutSpec.js [hunk 1]
```diff
@@ -58,11 +48,6 @@ describe('bpmn-auto-layout', function() {
     });
 
 
-    it('parallel flows', function() {
```
Comment (corocoto) [inline test/AutoLayoutSpec.js:61]: Removes this test scenario, cause it's body was empty

**Review by nikku — APPROVED** [review]
Looks great.

Will release as a major.

File: test/AutoLayoutSpec.js (hunk 1 above)
Comment (nikku) [reply test/AutoLayoutSpec.js:61]: :+1:

**Comment by corocoto** [comment]
@nikku, excuse me, but could you release this library in the nearest days? Because I want to use its new version for my project. Thanks!

**Pull request merged**